components/TimeStackedBarChart.vue

```diff
@@ -34,6 +34,7 @@
    "en": {
      "（注）同一の対象者について複数の検体を調査する場合あり": "(Note) More than one sample from the same subject may be tested.",
      "{date}の全体累計": "Cumulative total as of %{date}",
+     "{date}の合計": "On %{date}",
```

Comment:
https://docs.google.com/spreadsheets/d/1aksSfWIXE_L3rFYFjHz9xtVhh0n7ealMQ0eEz8GGPlw/edit#gid=0

Please check the translation table~